vpr/src/base/read_options.cpp

```diff
@@ -1480,7 +1480,9 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .help(
             "File format for the input atom-level circuit/netlist.\n"
```

Comment:
This comment looks wrong -- it's not the atom-level circuit/netlist but the arch file we're talking about here.